src/AddressRegistry.Api.Backoffice/Address/AddressCrabEditClient.cs

```diff
@@ -0,0 +1,199 @@
+                    Position = MapPosition(request.Position),
+                    PostalCode = request.PostalCode.AsIdentifier().Map(IdentifierMappings.PostalCode),
+                    Status = request.Status.AsIdentifier().Map(IdentifierMappings.AddressStatus),
+                    IsCorrection = true,
+                },
+                cancellationToken);
+
+            return CrabEditClientResult.From(updateResponse);
+        }
+
+        private AddressPosition MapPosition(Requests.AddressPosition position)
+        {
+            if (position == null)
+                throw new ArgumentNullException(nameof(position));
+
+            return new AddressPosition
+            {
+                Method = position.Method.AsIdentifier().Map(IdentifierMappings.PositionGeometryMethod),
+                Specification = position.Specification.AsIdentifier().Map(IdentifierMappings.PositionSpecification),
+                Wkt = _geoJsonMapper.ToWkt(position.Point)
+            };
+        }
+
+        public async Task<CrabEditClientResult> ChangeSubaddress(
+            int subaddressId,
+            ChangeAddressRequest request,
+            CancellationToken cancellationToken)
+        {
+            if (request == null)
+                throw new ArgumentNullException(nameof(request));
+
+            var updateResponse = await _client.Update(
+                new EditSubaddress
+                {
+                    SubaddressId = subaddressId,
+                    OfficiallyAssigned = request.OfficiallyAssigned,
+                    Position = MapPosition(request.Position),
+                    PostalCode = request.PostalCode.AsIdentifier().Map(IdentifierMappings.PostalCode),
+                    Status = request.Status.AsIdentifier().Map(IdentifierMappings.AddressStatus),
+                    IsCorrection = false,
+                },
+                cancellationToken);
+
+            return CrabEditClientResult.From(updateResponse);
+        }
+
+        public async Task<CrabEditClientResult> CorrectSubaddress(
+            int subaddressId,
+            CorrectAddressRequest request,
+            CancellationToken cancellationToken)
+        {
+            if (request == null)
+                throw new ArgumentNullException(nameof(request));
+
+            var updateResponse = await _client.Update(
+                new EditSubaddress
+                {
+                    SubaddressId = subaddressId,
+                    OfficiallyAssigned = request.OfficiallyAssigned,
+                    Position = MapPosition(request.Position),
+                    PostalCode = request.PostalCode.AsIdentifier().Map(IdentifierMappings.PostalCode),
+                    Status = request.Status.AsIdentifier().Map(IdentifierMappings.AddressStatus),
+                    IsCorrection = true,
+                },
+                cancellationToken);
+
+            return CrabEditClientResult.From(updateResponse);
+        }
```

Comment:
Same remark as above

Reply:
`ChangeAddressRequest ` and `CorrectAddressRequest` are different request models, you'd need a intermediate model to funnel it to 1 function. Change and Correct are 2 differnt business events in GRAR, they only happen to be exucuted with the same CRAB-edit function.